README.md

```diff
@@ -15,6 +15,297 @@ This repository contains two smart contracts:
+
+6. Add the organization to the `certs_wallet` in order to receive certificates.
+<a name="add_organization"></a>
+```
+soroban contract invoke \
+    --source-account <receiver-secret-key> \
+    --rpc-url https://rpc-futurenet.stellar.org:443 \
+    --network-passphrase 'Test SDF Future Network ; October 2022' \
+    --id <wallet_contract_id> \
+    -- add_organization \
+    --org_id 6f726731
+```
+7. The organization has already been added to the `Access Control List` of the wallet, then we can distribute a certificate from the `certs_governance`.
+<a name="distribute"></a>
+```
+soroban contract invoke \
+    --source-account SC54VFEAE3MXFJ2QYR5TCJDZYIYADAJFUPGZIDIEOFTAYQRMOVLRHM6U \
+    --rpc-url https://rpc-futurenet.stellar.org:443 \
+    --network-passphrase 'Test SDF Future Network ; October 2022' \
+    --id a13d49e0eb690e002940fe74e8d1304989a1e0f30218ec2317975925e7cfb17e \
+    -- distribute \
+    --admin <org-public-key> \
+    --receiver <receiver-public-key> \
+    --wallet_contract_id <wallet_contract_id>\
+    --cid 516d647479665459625653334b3969597142506a58786e346d624237614276456a59477a59576e7a52634d724543\
+    --distribution_date 1681414979
+```
+8. If at the moment of initializing the `certs_governance` contract in the `governance_rules` field we mark as `true`, we can revoke the certificate.
+<a name="revoke"></a>
+```
+  soroban contract invoke \
+    --source-account <org-secret-key> \
+    --rpc-url https://rpc-futurenet.stellar.org:443 \
+    --network-passphrase 'Test SDF Future Network ; October 2022' \
+    --id <governance_contract_id> \
+    -- revoke \
+    --admin <org-public-key> \
+    --holder <receiver-public-key> \
+    --wallet_contract_id <wallet_contract_id>
+```
+
+<a name="other_functions"></a>
+# Other functions
+
+## Certs Governane
+1. This function returns the name of the certificate
+```
+soroban contract invoke \
+    --source-account <org-secret-key> \
+    --rpc-url https://rpc-futurenet.stellar.org:443 \
+    --network-passphrase 'Test SDF Future Network ; October 2022' \
+    --id <governance_contract_id> \
+    -- name
+```
+2. This function returns whether the certificate is revocable or not.
+```
+soroban contract invoke \
+    --source-account <org-secret-key> \
+    --rpc-url https://rpc-futurenet.stellar.org:443 \
+    --network-passphrase 'Test SDF Future Network ; October 2022' \
+    --id <governance_contract_id> \
+    -- is_revocable
+```
+3. This function returns the expiration time of the certificate after its distribution.
```

Comment:
Actually this returns only the expiration time, used for calculating the expiration date during the distribution
```suggestion
3. This function returns the expiration time used to calculate the expiration_date.
```